torchci/lib/bot/ciflowPushTrigger.ts

```diff
@@ -215,6 +216,12 @@ async function handleLabelEvent(context: Context<"pull_request.labeled">) {
   }
 
   const prNum = context.payload.pull_request.number;
+  const owner = context.payload.repository.owner.login;
+  const repo = context.payload.repository.name;
+  // https://github.com/pytorch/pytorch/pull/26921 is a special PR that should
+  // never get ciflow tags
+  if (prNum == 26921 && isPyTorchPyTorch(owner, repo)) {
```

Comment:
do you need something inside this block?

Reply:
just return :)